packages/block-editor/src/components/link-control/index.js

```diff
@@ -98,7 +99,10 @@ import { DEFAULT_LINK_SETTINGS } from './constants';
  */
 
 const noop = () => {};
-
+const __newOnChangeForLinkControl = ( callback1, callback2, value ) => {
+	callback1( value );
+	callback2( value );
+};
```

Comment:
If we go with this route we can curry the function to pre-apply the handlers. That way you can just pass the new `value` to the curried function.

Reply:
refactored the `__newOnChangeForLinkControl` to call it just with the values in [91306d3](https://github.com/WordPress/gutenberg/pull/50401/commits/91306d3e80b07860f3782caf365f14bb26893af4).